packages/atlas/.gitignore

```diff
@@ -1 +1,2 @@
 figma-import.config.js
+queryFilter.json
```

Comment:
you added the file to `.gitignore`, but it's not actually ignored. I'm not sure if I understand what's happening here.